Test/ServerTelemetryChannel.Test/Shared.Tests/Implementation/TransmissionPolicyTest.cs

```diff
@@ -98,6 +102,76 @@ public void MaxStorageCapacityCanBeSetByPolicy()
             Assert.Equal(42, policy.MaxStorageCapacity);
         }
 
+        [TestClass]
+        public class GetBackOffTime : ErrorHandlingTransmissionPolicyTest
+        {
+            [TestMethod]
+            public void NoErrorDelayIsSameAsSlotDelay()
+            {
+                var policy = new TestableTransmissionPolicy();
+                TimeSpan delay = policy.GetBackOffTime(new WebHeaderCollection());
+                Assert.Equal(TimeSpan.FromSeconds(10), delay);
+            }
+
+            [TestMethod]
+            public void FirstErrorDelayIsSameAsSlotDelay()
+            {
+                var policy = new TestableTransmissionPolicy();
+                policy.ConsecutiveErrors = 1;
+                TimeSpan delay = policy.GetBackOffTime(new WebHeaderCollection());
+                Assert.Equal(TimeSpan.FromSeconds(10), delay);
+            }
+
+            [TestMethod]
+            public void UpperBoundOfDelayIsMaxDelay()
+            {
+                var policy = new TestableTransmissionPolicy();
+                policy.ConsecutiveErrors = int.MaxValue;
+                TimeSpan delay = policy.GetBackOffTime(new WebHeaderCollection());
+                Assert.InRange(delay, TimeSpan.FromSeconds(10), TimeSpan.FromSeconds(3600));
```

Comment:
We check that it is not more than this interval. Since we use Random for generation it is best we can do.